Script Analysis Examples/Specification Analysis/Simple Specification Analysis Example.ipynb

```diff
@@ -0,0 +1,245 @@
+    "\n",
+    "The SYSTEMLINK_API_KEY environment variable specifies an API key created for the user executing this notebook, which provides Role Based Access Control to the various SystemLink APIs called by this notebook. The API key will expire after 24 hours.\n",
+    "The SYSTEMLINK_HTTP_URI environment variable gives the base URL to the SystemLink instance executing this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import scrapbook as sb\n",
+    "\n",
+    "from nisystemlink.clients.core import HttpConfiguration\n",
+    "from nisystemlink.clients.spec import SpecClient\n",
+    "from nisystemlink.clients.spec.models import QuerySpecificationsRequest, UpdateSpecificationsRequest\n",
+    "\n",
+    "api_key = os.getenv(\"SYSTEMLINK_API_KEY\")\n",
+    "systemlink_uri = os.getenv(\"SYSTEMLINK_HTTP_URI\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Parameters\n",
+    "\n",
+    "These are the parameters that the notebook expects to be passed in by SystemLink. For notebooks designed to be triggered by pressing the 'Analyze' button in SystemLink Specs grid inside product details page, they must tag the cell with 'parameters' and at minimum specify the following in the cell metadata using the JupyterLab Property Inspector (double gear icon):\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "  \"papermill\": {\n",
+    "    \"parameters\": {\n",
+    "      \"spec_ids\": [],\n",
+    "      \"product_id\": \"\"\n",
+    "    }\n",
+    "  },\n",
+    "  \"systemlink\": {\n",
+    "    \"namespaces\": [],\n",
+    "    \"parameters\": [\n",
+    "      {\n",
+    "        \"display_name\": \"spec_ids\",\n",
+    "        \"id\": \"spec_ids\",\n",
+    "        \"type\": \"string[]\"\n",
+    "      },\n",
+    "      {\n",
+    "        \"display_name\": \"product_id\",\n",
+    "        \"id\": \"product_id\",\n",
+    "        \"type\": \"string\"\n",
+    "      }\n",
+    "    ],\n",
+    "    \"version\": 2\n",
+    "  },\n",
+    "  \"tags\": [\"parameters\"]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "For more information on how parameterization works, review the [papermill documentation](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#how-parameters-work).\"\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec_ids = []\n",
+    "product_id = \"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "server_configuration = HttpConfiguration(\n",
```

Comment:
We should be able to use `JupyterHttpConfiguration` here. This will pull the server URL and the API key from the appropriate environment variables. When we make this update, we should also remove the API key and url variables from the code above.

Reply:
Updated to use JupyterHttpConfiguration